public/raw-salary.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Raw Salary Ranges</title>
+</head>
+<body>
+  <canvas id="salarychart" width="400" height="200"></canvas>
+  <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.2/Chart.js"></script>
+  <script>
+    const salaryJSON = [{"salary":40000},{"salary":50000},{"salary":50000},{"salary":50000},{"salary":60000},{"salary":60000},{"salary":60000},{"salary":60000},{"salary":60000},{"salary":60000},{"salary":60000},{"salary":60000},{"salary":60000},{"salary":60000},{"salary":70000},{"salary":70000},{"salary":70000},{"salary":70000},{"salary":70000},{"salary":70000},{"salary":70000},{"salary":70000},{"salary":70000},{"salary":70000},{"salary":70000},{"salary":80000},{"salary":80000},{"salary":80000},{"salary":80000},{"salary":80000},{"salary":80000},{"salary":80000},{"salary":80000},{"salary":80000},{"salary":90000},{"salary":90000},{"salary":90000},{"salary":90000},{"salary":90000},{"salary":90000},{"salary":100000},{"salary":100000},{"salary":100000},{"salary":100000},{"salary":100000},{"salary":100000},{"salary":100000},{"salary":100000},{"salary":100000},{"salary":110000},{"salary":110000},{"salary":110000},{"salary":110000},{"salary":110000},{"salary":110000},{"salary":110000},{"salary":110000},{"salary":110000},{"salary":110000},{"salary":110000},{"salary":120000},{"salary":120000},{"salary":120000},{"salary":120000},{"salary":120000},{"salary":120000},{"salary":130000},{"salary":130000},{"salary":130000},{"salary":130000},{"salary":140000},{"salary":140000},{"salary":140000},{"salary":140000},{"salary":140000},{"salary":150000},{"salary":160000},{"salary":160000},{"salary":170000},{"salary":170000},{"salary":180000}]
+
+    const chartHTML = document.getElementById('salarychart')
+
+    const data = salaryJSON.map(salary => salary.salary)
```

Comment:
Just because I would have to manually extract the salaries into an array beforehand. 

It's not the most performant, but I wanted to make reusable functions that I could just stick into any possible chart configuration with minor changes of names, data, etc..

Reply:
That makes perfect sense.  Thanks!